src/ValidatorKeys.cpp

```diff
@@ -46,9 +46,9 @@ ValidatorKeys::ValidatorKeys (KeyType const& keyType)
     : keyType_ (keyType)
     , tokenSequence_ (0)
     , revoked_ (false)
+    , publicKey_ (generateKeyPair (keyType_, randomSeed ()).first)
+    , secretKey_ (generateKeyPair (keyType_, randomSeed ()).second)
```

Comment:
I think you should put the return value of `generateKeyPair(keyType_, randomSeed())` into a default argument and then move the pieces in these member initializers.